addons/website_form/data/website_form_data.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
```

Comment:
imo all this part and the basic page should be in website and not website_form. (as previously)